pkg/scheduler/api/types.go

```diff
@@ -326,3 +326,14 @@ func (h HostPriorityList) Less(i, j int) bool {
 func (h HostPriorityList) Swap(i, j int) {
 	h[i], h[j] = h[j], h[i]
 }
+
+// ScheduleResult represents the result of one pod scheduled. It will contain
+// the final selected Node, along with the selected intermediate information.
+type ScheduleResult struct {
+	// Name of the scheduler suggest host
+	SuggestedHost string
+	// Number of nodes scheduler evaluated on one pod scheduled
+	EvaluatedNodesNumber int
```

Comment:
Since this is going in "api", I don't think it makes sense to put "Number" in th ename.

Reply:
It has been modified.